apps/website-proxy/src/nginx.template.conf

```diff
@@ -49,6 +49,12 @@ http {
             proxy_pass https://website-25-production.k8s.bluedot.org;
         }
 
+        # Redirect 404 errors to the same route on the new website
+        error_page 404 = @website_25_404;
+        location @website_25_404 {
+            proxy_pass https://website-25-production.k8s.bluedot.org$request_uri;
+        }
+
         # Default all other traffic to old site
```

Comment:
Quick note on testing this *locally*:

1. Replace `docker run` line in `libraries/docker-scripts/src/start.sh` with `docker run --rm --publish 8001:8080 $IMAGE_ID` so we can run `website-25` on `8000`.
2. Replace `https://website-25-production.k8s.bluedot.org` with `http://host.docker.internal:8000` to use your local version of `website-25` instead of `website-25-production`.
3. Start `website-proxy` via `npm start`.
4. Start `website-25` via `npm start`.
5. Use `curl` to trigger a request with the correct `Host`, e.g. `curl -vv localhost:8001/does-not-exist --header 'Host: bluedot.org'`
6. Check. the logs :)

Reply:
Great call, @tarinrickett! 

@Will-Howard: Would you mind adding (sth like 😇) this to the `website-proxy/README.md` once you've gotten a chance to play around with it?